docs/reference/data-streams/logs.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[logsds]]
+== Logs data stream
+
+You can use a logs data stream to store log data more efficiently. In our benchmarks,
+log data stored in a logs data stream used ~2.5 times less disk space than a regular data
+stream. The exact impact will vary per data set. The main benefit from using a logs data stream is reduced storage.
+
+The following features are enabled in a logs data stream:
+
+* <<synthetic-source,Synthetic source>>, which omits storing the _source. When _source or part of it is requested it is synthesized on the fly at runtime.
+
+* Index sorting. By default indices are sorted by `host.name` and `@timestamp` fields at index time, which can be overwritten.
+
+* Enable more space efficient compression for fields with doc values enabled.
```

Comment:
we already say the feature is enabled in the intro to the list

i'm also interested in the "why" behind this - what makes us able to more efficiently compress these fields in this context? 

```suggestion
* More space efficient compression for fields with <<doc-values,`doc_values`>> enabled.
```